src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const kindOfAction of actions) {
+    switch (kindOfAction.type) {
+      case 'addProperties':
+        Object.assign(state, kindOfAction.extraData);
+        break;
+      case 'removeProperties':
+        kindOfAction.keysToRemove.forEach(el => {
+          delete state[el];
+        });
```

Comment:
why have we iterated here through `forEach`, but our `actions` through `for of`?

Reply:
Why not? 
If there is some restriction about it? 
Why couldn't we use both of them? 🤷‍♂️
👌 fixed

Reply:
@YeMyr there is no restriction, as for me, it degrades the readability of our code if we can iterate with forEach, it would be better to use this whenever possible, unless it requires additional conversions